com.mudeprolinux.yml

```diff
@@ -0,0 +1,33 @@
+app-id: com.mudeprolinux.whakarere
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: whakarere
+
+finish-args:
+  # X11 and IPC access
+  - --socket=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # PulseAudio
+  - --socket=pulseaudio
+
+modules:
+  - name: Whakarere
+    buildsystem: simple
+    build-commands:
+      - install -D whakarere.py /app/bin/whakarere
+      - install -Dm644 whakarere.png /app/share/icons/hicolor/512x512/apps/com.mudeprolinux.whakarere.png
+      - install -Dm644 whakarere.desktop /app/share/applications/com.mudeprolinux.whakarere.desktop
+    sources:
+      - type: file
+        path: https://raw.githubusercontent.com/tobagin/whakarere/main/whakarere.py
```

Comment:
this is not how you specify to use a git repository.

also NEVER reference a branch. you need to reference a specific tag/commit